src/Cake.Common/Tools/MSBuild/MSBuildResolver.cs

```diff
@@ -95,7 +95,8 @@ private static DirectoryPath GetVisualStudioPath(ICakeEnvironment environment, M
             {
                 "Enterprise",
                 "Professional",
-                "Community"
+                "Community",
+                "BuildTools"
             };
 
             var visualStudio2017Path = environment.GetSpecialPath(SpecialPath.ProgramFilesX86);
```

Comment:
@Trass3r yes, out of the box, Cake handles the default installation paths, and for anything else, there is the VSWhere Tool, which can then be used to set ToolPath on tools that require it, as described here:

http://cakebuild.net/blog/2017/03/vswhere-and-visual-studio-2017-support